lib/dfid-transition/patch/specialist_publisher/regions.rb

```diff
@@ -0,0 +1,37 @@
+require 'dfid-transition/patch/specialist_publisher/base'
+require 'rest-client'
+require 'nokogiri'
+
+module DfidTransition
+  R4D_ADVANCED_SEARCH = "http://r4d.dfid.gov.uk/Search/SearchResearchDatabase.aspx"
+
+  module Patch
+    module SpecialistPublisher
+      class Regions < Base
+        def mutate_schema
+          html = RestClient.get R4D_ADVANCED_SEARCH
+
+          region_facet['allowed_values'] =
+            transform_to_label_value(html).reject { |lv| lv[:value] == "==" }
+        end
+
+      private
+
+        def region_facet
+          facet('region')
+        end
+
+        def transform_to_label_value(html)
+          doc = Nokogiri::HTML(html)
+          options = doc.css('#ContentPlaceHolderMainDiv_ddlRegionList option')
+          options.map do |option|
+            {
+              label: option.xpath('./text()'),
```

Comment:
Oh, this is odd. We boshed that commit that replaced this with `.text` because it was already on master, thinking it was a no-op. Clearly it was a some-op, as `xpath` has returned. Still does the same thing, mind, just more verbosely.